Daramola98/WEconnectApp
# Connects Sign Up, Login, User Profile Business Listing components to Redux Store

#### What does this PR do?
This PR enables users to successfully register, login and view businesses.

#### Description of Task to be completed?
- Setup Redux Store
- Setup Action Creators and Reducers
- Connect Components to redux store
- Sign Up Feature
- Login Feature
- View Business Listing Feature
-  View User Profile Feature

#### How should this be manually tested?
View application at weconnect-api.herokuapp.com
Success or failure when user tries to sign up or login
User should be redirected to user profile page after successfull sign up or login
Navigate to Business Listing Page to view all businesses

What are the relevant pivotal tracker stories?
#156776093
#156776051
#156776044
